README.md

```diff
@@ -42,51 +42,59 @@ Your code interview will be evaluated based on your repo, so make sure all files
+### Python Task 4: Spatial data processing
 
-Depending on how you generate the map, you can store the map as a `.png` or `.html` under the `data` folder.
+At Data Engineering, we enhance datasets with geospatial attributes, such as point locations and administrative boundaries. To help us better understand the data from **Python Task 1**, we would like you to join the initial raw data to the some NYC administraive boundary of your chosing. Then create a choropleth map of count of complaints of a specific `complaint_type` of your choice.
 
-### Task 5: SQL
+Depending on how you generate the map, you can store the map as a `.png` or `.html` under the `data` folder. Please add a title that accurately tells the reader what they are looking at.
 
-We ❤️ SQL! At Data Engineering, we deal with databases a lot and we write a lot of fast and simple ETL pipelines using SQL. In this task, you will:
 
-- Load the `data/raw.csv` into a database of your choice and name the table `sample_311`. Make sure this process is captured in a script.
-- Perform the same aggregation in **Task 2** in SQL and store the results in a table (same name as the corresponding csv file).
+### SQL/Docker Task 1: Build container and load 
+
+We ❤️ SQL and docker! At Data Engineering, we work with database containers a lot and we write a lot of fast and simple ETL pipelines using SQL. In this task, you will:
 
-> Note: Depending on your preference, you can use or [Postgres](https://www.postgresql.org/), which is prefered; however, if you are familiar with [SQLite](https://docs.python.org/3/library/sqlite3.html) (much easier to set up and use), you can use that too.
+- Set up POSTGIS container using an image
+- Load the `data/raw.csv` into the database and name the table `sample_311`. Make sure this process is captured in a script.
+- Perform the same aggregation in **Task 2** in SQL and store the results in a table (same name as the corresponding csv file).
 
-### Task 6: Spatial SQL
+### SQL/Docker Task 2: Spatial SQL
 
 A lot of popular databases have geospatial extensions, which makes spatial data processing in SQL super easy to use. In this task you will:
 
-- Load the NTA data to the database as a spatial table
-- Do a spatial join in SQL between `sample_311` and the NTA table and add a `nta` column to `sample_311`
+- Load the adminstrative data you used in python task 4 to the database as a spatial table
+- Do a spatial join in SQL between `sample_311` and the administrative boundaries and add a corresponding column to `sample_311`
```

Comment:
Is the corresponding the unique ID of the admin boundary?  If so, I'd explicitly say that.

Reply:
Good question. I think we could say "administrative boundary ID" which is easier or "foreign key" to test their knowledge of SQL terms/googling ability